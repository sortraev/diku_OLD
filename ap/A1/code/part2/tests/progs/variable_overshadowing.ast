[SDef "x" (Const (IntVal 42)),
 SExp (Call "print" [Var "x"]),
 SDef "y" (Const (IntVal (-1337))),
 SExp (Call "print" [Var "y"]),
 SDef "x" (Const (IntVal 4283)),
 SExp (Call "print" [Var "x"])]
