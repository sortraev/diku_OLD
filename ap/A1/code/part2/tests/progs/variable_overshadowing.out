42
-1337
4283
